Firestore/core/src/firebase/firestore/util/status.cc

```diff
@@ -0,0 +1,132 @@
+// TODO(rsgowman): how should the copyright be adapted?
```

Comment:
This is Google code, being released by Google as part of Firestore. That it was previously open sourced isn't material.

I'd do like we did here: 

https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/core/src/firebase/firestore/util/iterator_adaptors.h#L2

See the differences between:
https://github.com/google/protobuf/blob/master/src/google/protobuf/stubs/status.h
https://github.com/grpc/grpc/blob/master/include/grpcpp/impl/codegen/status.h
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/lib/core/status.h

Reply:
Regardless of the copyright content, this is in `.cc` file. Isn't copyright in `.h` file enough, is it.

Reply:
Every file must have a copyright notice.